### ISSUE TITLE
Add User Feedback protocol classes

### ISSUE THREAD

**Issue by GotenXiao** [getsentry/sentry-dotnet-protocol#48]
At present, this library does not implement the [User Feedback](https://docs.sentry.io/api/projects/post-project-user-reports/) models or wire protocol, which is blocking getsentry/sentry-dotnet#153.

It's not entirely clear from the Sentry API documentation whether the User Feedback endpoint supports DSN-based authentication, or indeed if it requires the client to make backend requests against the full Sentry API, with knowledge of the organisation and project slugs, and an API key, or if there's a way to use the regular event submission endpoint.

**Comment by bruno-garcia** [comment]
That's right that currectly we can't send user-feedback with DSN based auth. This is something we need to work on. We might be introducing a new format to send data from SDKs to the backend which would be addressing this issue. Otherwise we'll need to add another user-endpoint to Sentry.
